cmd/tools/cwl/cwl.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
```

Comment:
Oh wow, you even made your own CWL :)

We should probably combine these two into one tool in the future, but this is good for now.